README.md

```diff
@@ -165,6 +165,7 @@ A curated list of resources for R Shiny.  (*Featured on [Awesome-R](https://awes
 - [reactlog](https://github.com/rstudio/reactlog) - Visual insight into a Shiny apps reactive graph.
 - [supreme](https://github.com/strboul/supreme) - Structure Shiny applications developed with modules.
 - [shinymanager](https://github.com/datastorm-open/shinymanager/) - Simple and secure authentication for single 'Shiny' applications using a SQLite database.
+- [polished](https://github.com/Tychobra/polished) - Authentication, user administration, and web hosting for secure Shiny apps & Rmarkdown documents (flexdashboards, HTML, & PDF)
```

Comment:
```suggestion
- [polished](https://github.com/Tychobra/polished) - Authentication, user administration, and web hosting for secure Shiny apps & Rmarkdown documents (flexdashboards, HTML, & PDF).
```

Reply:
Just adding a period here @phoward38 ! It looks great.

Reply:
@sdhutchins Awesome, thanks!